### ISSUE TITLE
JSX Optional Chaining results in Unexpected Token error on compilation

### ISSUE THREAD

**Issue by jbaldwinson** [failwyn/WebCompiler#58]
### Installed product versions
- Visual Studio: 17.4.1
- This extension: 1.14.8

### Description
Added a null check chaining operator into a JSX file, and can no longer compile into JS. I think this is due to the version of Babel being targeted. 

### Steps to recreate
1. Have a JSX file
2. Add a null chaining operator
3. try to compile the jsx file
4. Get the error

**Comment by failwyn** [comment]
I can take another look into upgrading Babel, but last time I tried, it caused other issues and I ended up having to revert back to the old version.

https://github.com/failwyn/WebCompiler/issues/15

**Comment by jbaldwinson** [comment]
Yeah, I saw that commit and rollback. Happy to help in anyway I can.
